test/command_line/test_generate_mask.py

```diff
@@ -171,3 +171,23 @@ def test_generate_mask_trusted_range(dials_data, tmpdir):
 
     # Overloads should not be included in the mask
     assert (mask1[0] == mask2[0]).all_eq(True)
+
+
+def test_generate_whole_panel_mask(input_experiment_list, tmpdir):
+    params = phil_scope.fetch(
+        phil.parse(
+            """
+untrusted {
+  panel = 0
```

Comment:
Is this a multi-panel test case? If so it might be better to test with a non-zero panel id?

Reply:
The test is single panel, but in this case 0 specifically was tripping me up (see a6fa94b) so that's why it's tested.